lib/config.js

```diff
@@ -41,7 +42,8 @@ const helpMessage = `Usage:
   -r, --replace                 Replace the input IPA file with the resigned one
   -u, --unfair                  Resign encrypted applications
   -w, --without-watchapp        Remove the WatchApp from the IPA before resigning
-  -W, --without-xctests         Remove the XCTests from the resigned IPA
```

Comment:
i think we can split out those 3 options into a separate section in the help, like it's done for entitlemnets, infoplist..

Reply:
i would name it "Stripping"

Reply:
Done!